server/service.go

```diff
@@ -498,11 +498,15 @@ func (m *BoostService) processBellatrixPayload(w http.ResponseWriter, req *http.
 		go func(relay RelayEntry) {
 			defer wg.Done()
 			url := relay.GetURI(pathGetPayload)
+
 			log := log.WithField("url", url)
 			log.Debug("calling getPayload")
+			errorHandler := func(err error) {
+				log.WithError(err).Debug("error making request to relay, retrying")
```

Comment:
this should probably be an error log, at least warn, so users see it
```suggestion
				log.WithError(err).Warn("error making request to relay, retrying")
```